docs/en/observability/user-experience-visualizations.asciidoc

```diff
@@ -3,64 +3,82 @@
 
 experimental[]
 
-Based on the {user-experience} data from your instrumented applications, you can create various
-detailed visualizations for performance distributions of your web applications or a
-key performance indicator (KPI) over time, such as page load times, backend times, and many others.
+Based on the {user-experience} data from your instrumented applications, you can create
+detailed visualizations for performance distributions, key performance indicators (KPI) over time,
+and for core web vitals of your web applications.
 
 [role="screenshot"]
 image::images/user-experience-visualization.png[User experience visualization]
 
-. Go to *Observability > {user-experience}*.
+. Go to *Observability > {user-experience} > Dashboard*.
 . Click *Analyze data*.
 . For data type, select *{user-experience} (RUM)*.
 . Under **Report**, choose the type of data you want to analyze:
 +
-|=== 
+|===
+
+| *KPI over time* | The KPI over time histogram represents the performance indicators based on
+the metric you select.
+
+By default, the `page views` metric is selected. Hover over the chart to display crosshairs
+with specific metric data. 
 
 | *Performance distribution* | The Performance distribution time-series chart enables you to examine the perceived
 performance of your web applications based on the metric you select.
 
 By default, the `page load time` metric is selected. Hover over the chart to display crosshairs with specific metric data.
 
-| *KPI over time* | The KPI over time histogram is a graphical representation of the performance indicators based on
-the metric you select.
+| *Core web vitals* | The Core web vitals chart is a graphical representation of key metrics, such as
+loading performance, load responsiveness, and visual stability, for each of your web applications.
 
-By default, the `page views` metric is selected. Hover over the chart to display crosshairs
-with specific metric data.
+By default, the `largest contentful paint` metric is selected. Hover over the chart to display crosshairs
+with performance indicators for each web application: `poor`, `average`, and `good`.
 
 |===
 
 . Now let's define your report. Set the time filter as an absolute or relative time.
 . Select the web application name and its environment. You can make multiple selections.
-. Form the list of available metrics, select the metric you want to analyze:
+. From the *Metric* dropdown, choose the metric you want to analyze (which are dependent on which report
+you previously selected):
```

Comment:
This sounds funky to me. What about something like this?

> From the *Metric* dropdown, choose the metric you want to analyze; available options depend on the previously selected report type.

Or maybe

> From the *Metric* dropdown, choose the metric you want to analyze; different options are available for each report type.

Idk. Maybe I'm overthinking it 😁. If you do change it, hit the other one a few lines down too.

Reply:
Great suggestions. I'll go with your first one. Thanks!